README.md

```diff
@@ -39,6 +39,7 @@ Contributions always welcome!
   - `isNegative()`
 - `isNumeric()`
 - ⭐ `isObject()` if it's a js plain Object
+- `isPastDate()`
```

Comment:
should we have groups in the readme? otherwise it's on line 30

Reply:
Oh, I didn't see it. Removed.